tests/brun/apply-00.txt

```diff
@@ -1,3 +1,3 @@
-brun -c '(a (q . +) (q . (3 4 5)))'
-cost = 84
-12
+brun -c '(a (q . 2) (q . (3 4 5)))'
```

Comment:
I'm of two mind using the opcodes vs the keyword here.

It's less ambiguous for consensus critical interpretation of clvm byte code to use the opcodes.

But it makes it harder to read the test. Hmm, that gives me an idea for a small set of new tests.

Reply:
The change here is to make explicit that `+` is no longer treated as an operator, but a path.